netket/operator/_sumoperators.py

```diff
@@ -0,0 +1,41 @@
+from typing import Optional, List
+
+from netket.utils.types import DType
+
+from netket.hilbert import AbstractHilbert
+from netket.operator import ContinousOperator
+
+import jax.numpy as jnp
+
+
+class SumOperator(ContinousOperator):
+    def __init__(
+        self,
+        hilbert: AbstractHilbert,
+        operators: List,
+        coeff: float,
+        dtype: Optional[DType] = float,
+    ):
+        r"""Args:
+        operators: A list of ContinousOperator objects
```

Comment:
please document all the fields, not only a subset

Reply:
yes